### PR TITLE
Add a couple more raising cases (fix #36)

### LINKED ISSUE #36: Raise something more explicit when wrong Docker folder is passed
Something along the lines:

```
diff --git a/docker_explorer/lib/container.py b/docker_explorer/lib/container.py
index 5f926e1..5a7df25 100644
--- a/docker_explorer/lib/container.py
+++ b/docker_explorer/lib/container.py
@@ -81,8 +81,14 @@ class Container(object):
     container_info_json_path = os.path.join(
         self.docker_directory, 'containers', container_id,
         self.container_config_filename)
-    with open(container_info_json_path) as container_info_json_file:
-      container_info_dict = json.load(container_info_json_file)
+    try:
+      with open(container_info_json_path) as container_info_json_file:
+        container_info_dict = json.load(container_info_json_file)
+    except IOError as exception:
+      raise errors.BadContainerException(
+          'Unable to open configuration file {0} make sure you use the proper docker dir'.format(
+              container_info_json_path)
+      )
 
     if container_info_dict is None:
       raise errors.BadContainerException(
```

### PR DESCRIPTION
**Pull request by rgayon**

### CODE REVIEW

File: docker_explorer/de.py [hunk 1]
```diff
@@ -156,14 +153,23 @@ def GetAllContainers(self):
...
+    if not os.path.isdir(self.containers_directory):
+      raise errors.BadStorageException(
+          'Containers directory {0} does not exist'.format(
+              self.containers_directory))
     container_ids_list = os.listdir(self.containers_directory)
     if not container_ids_list:
-      print('Couldn\'t find any container configuration file (\'{0:s}\'). '
-            'Make sure the docker repository ({1:s}) is correct. '
+      print('Could not find container configuration files ({0:s}) in {1:s}.\n'
+            'Make sure the docker repository ({2:s}) is correct.\n'
```
Comment (tomchop) [inline docker_explorer/de.py]: Sounds like you mean `docker directory` instead of `repository`?

File: docker_explorer/de.py [hunk 2]
```diff
@@ -52,11 +52,8 @@ def _SetDockerDirectory(self, docker_path):
     """
     self.docker_directory = docker_path
     if not os.path.isdir(self.docker_directory):
-      err_message = (
-          '{0:s} is not a Docker directory\n'
-          'Please specify the Docker\'s directory path.\n'
-          'hint: de.py -r /var/lib/docker').format(self.docker_directory)
-      raise errors.BadStorageException(err_message)
+      msg = '{0:s} is not a Docker directory\n'.format(self.docker_directory)
```
Comment (tomchop) [inline docker_explorer/de.py]: Just FYI, there's a `\n` here which is absent from the other exceptions.

File: docker_explorer/de.py [hunk 3]
```diff
@@ -301,4 +314,9 @@ def Main(self):
 
 
 if __name__ == '__main__':
-  DockerExplorer().Main()
+  try:
+    DockerExplorer().Main()
+  except errors.BadStorageException as exc:
+    print('ERROR: {0}\n'.format(exc.message))
+    print('Please specify a proper Docker\'s directory path.\n'
```
Comment (tomchop) [inline docker_explorer/de.py]: `proper Docker directory path`

File: docker_explorer/de.py (hunk 1 above)
Comment (rgayon) [reply docker_explorer/de.py]: Fixed

File: docker_explorer/de.py (hunk 2 above)
Comment (rgayon) [reply docker_explorer/de.py]: done

File: docker_explorer/de.py (hunk 3 above)
Comment (rgayon) [reply docker_explorer/de.py]: done

**Pull request merged**